pkg/roachprod/roachprod.go

```diff
@@ -1539,18 +1539,17 @@ func RemoveLabels(l *logger.Logger, clusterName string, labels []string) error {
 
 // Create TODO
 func Create(
-	ctx context.Context,
-	l *logger.Logger,
-	username string,
-	numNodes int,
-	createVMOpts vm.CreateOpts,
-	providerOptsContainer vm.ProviderOptionsContainer,
+	ctx context.Context, l *logger.Logger, username string, opts []cloud.ClusterCreateOpts,
```

Comment:
`opts ...cloud.ClusterCreateOpts` may make it easy for places where we just have one ClusterCreateOpts to pass

Reply:
Good catch, fixed